src/search/ipfs-search-table/IPFSSearchTable.js

```diff
@@ -0,0 +1,153 @@
+import React, { useState } from 'react'
+// import PropTypes from 'prop-types'
+import { connect } from 'redux-bundler-react'
+import { withTranslation } from 'react-i18next'
+import { Table, Column, AutoSizer, SortDirection } from 'react-virtualized'
+import { sortByProperty } from '../../lib/sort'
+// import { getResourceURL } from '../../helpers/ApiSearchHelper'
+import IPFSSearchLogoHex from '../../icons/IPFSSearchLogoHex'
+import FileIcon from '../../files/file-icon/FileIcon'
+import StrokeIpld from '../../icons/StrokeIpld'
+import typeFromExt from '../../files/type-from-ext'
+// import GlyphLinkExternal from '../../icons/GlyphLinkExternal'
+import './IPFSSearchTable.css'
+
+const IPFSSearchTable = ({ t, searchResults }) => {
+  const [sortBy, setSortBy] = useState('')
+  const [sortDirection, setSortDirection] = useState(SortDirection.ASC)
+  const sortedList = (searchResults || []).sort(sortByProperty(sortBy, sortDirection === SortDirection.ASC ? 1 : -1))
+  const tableHeight = 500
+
+  const sort = ({ sortBy, sortDirection }) => {
+    setSortBy(sortBy)
+    setSortDirection(sortDirection)
+  }
+
+  const typesQueryLookup = {
+    audio: 'audio',
+    image: 'images',
+    video: 'video',
+    text: 'text',
+    html: 'text',
+    pdf: 'text',
+    directory: 'directories'
+  }
+
+  const getQueryFileType = (type, title, mimetype) => {
+    if (type === 'directory') {
+      return 'directories'
+    }
+    if (type === 'unknown') {
+      return 'other'
+    }
+
+    let strippedTitle = new DOMParser().parseFromString(title, 'text/html')
+    strippedTitle = strippedTitle.body.textContent || ''
+
+    const fileTypeFromExt = typeFromExt(strippedTitle)
+    const queryTypeString = typesQueryLookup[mimetype?.split('/')[0]] || typesQueryLookup[fileTypeFromExt] || 'other'
```

Comment:
does ipfs-search really return html that we need to parse in order to get data? I feel like this should be more simple. Are they aware of this?